docs/manual/install.qmd

```diff
@@ -0,0 +1,89 @@
+# Installation
+
+The commands below can be entered using any terminal application. Search for
+"terminal" or "command line" in your OS application menu.
+
+**Attention**: the python and pip executables may have different names
```

Comment:
good idea. done.

Reply:
this may break the pdf generation. I added a note for how to fix it on linux, but it may work out of the box on windows. Please try generating it again, @yulric.

Reply:
The PDF looks good on my Mac.

Reply:
great, should be good then.